app/views/layouts/application.html.erb

```diff
@@ -25,10 +25,18 @@
   <body class="mainstream <%= yield :body_classes %>">
     <div id="wrapper" class="<%= wrapper_class(@publication || @presenter) %>">
       <% unless current_page?(root_path) %>
-        <%= render partial: 'govuk_component/breadcrumbs',
-          locals: {
-            breadcrumbs: breadcrumbs[:breadcrumbs]
-          } %>
+        <% if @navigation.should_present_taxonomy_navigation? %>
```

Comment:
Is it weird that we show breadcrumbs differently between the two styles?

Reply:
Yep. I want us to  look into this. Hoping that collapse on mobile can be the default too.

Reply:
Sure, so long as this is the intended behaviour for now.